### PR TITLE
Update TGS DMAPI to 7.0.0

### PR DESCRIPTION
**Pull request by Cyberboss**

### CODE REVIEW

File: code/modules/tgs/core/datum.dm
Comment (AffectedArc07) [inline code/modules/tgs/core/datum.dm:11]: This file got renamed to avoid files with the same name in the codebase

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]
Same applies with the other files - I'll manually do the update if you want

File: code/modules/tgs/core/datum.dm
Comment (Cyberboss) [reply code/modules/tgs/core/datum.dm:11]: You better take care of it then.

Comment (Cyberboss) [reply code/modules/tgs/core/datum.dm:11]: Reminder, this branch has maintainer edits enabled

**Comment by ike709** [comment]
OpenDream TGS server real

**Comment by S34NW** [comment]
Removed the on hold label, known contributor.

File: code/modules/tgs/core/datum.dm
Comment (AffectedArc07) [reply code/modules/tgs/core/datum.dm:11]: > Reminder, this branch has maintainer edits enabled

Oh I didnt see this and already made another PR

**Comment by S34NW** [comment]
image
Affected is not very competent despite his promises, we'll go with https://github.com/ParadiseSS13/Paradise/pull/23657 seeing as he's already touched up the things he requested changed in this one.

Thanks for doing the leg work on this update.

**Pull request closed without merging**
